House-of-glass/src/components/Header/Header.jsx

```diff
@@ -1,31 +1,52 @@
-import React from 'react'
-import { Link } from 'react-router-dom'
-import "./Header.css"
-import { useAuth } from 'Utils/AuthContext'
-
+import React from "react";
+import { Link } from "react-router-dom";
+import "./Header.css";
+import { useAuth } from "Contexts/Auth/AuthContext";
 
 export default function Header() {
-  const {userDetails}= useAuth();
-  const {cartList, wishList} = userDetails;
-    return (
-      <header className="glass-navbar">
-          <Link to="/" className="glass-logo">
-            <span id="hamburger_open" className="material-icons icon--sidebar">menu</span>
-            <span id="hamburger_close" className="material-icons icon--sidebar">menu_open</span>
-            <h3 className="glass-hero-text header-5">House Of Glass</h3>
+  const { userDetails } = useAuth();
+  const { cartList, wishList } = userDetails;
+  return (
+    <header className="glass-navbar">
+      <Link to="/" className="glass-logo">
+        <span id="hamburger_open" className="material-icons icon--sidebar">
+          menu
+        </span>
+        <span id="hamburger_close" className="material-icons icon--sidebar">
+          menu_open
+        </span>
+        <h3 className="glass-hero-text header-5">House Of Glass</h3>
+      </Link>
+      <div className="glass-search input__container">
+        <input
+          type="search"
+          className="input__field glass-search-input br-24 position-relative"
+          placeholder="What today?"
+          name="searchbar"
+          id="search-bar"
```

Comment:
Put auto complete on attribute

Reply:
Yes, I missed this on my end. Thank You.